packages/markers/src/browser/problem/problem-utils.ts

```diff
@@ -0,0 +1,48 @@
+/********************************************************************************
+ * Copyright (C) 2020 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { Marker } from '../../common/marker';
+import { Diagnostic } from '@theia/languages/lib/browser';
+
+export namespace ProblemUtils {
```

Comment:
(nit) I would better to add them as static method to `ProblemTree`. `Utils` always feels like it is useful to share but I don't know what is a good concept for it.